### ISSUE TITLE
Tello drone shuts down when idle

### ISSUE THREAD

**Issue by anoop4** [dji-sdk/Tello-Python#30]
Hi,

I have my Tello drone sitting idle on my desk and it shuts down after about 4 minutes.
This happens consistently. 

Does anyone know if this is a common issue? Is it due to overheating of the downward-facing sensors that are close to the table surface?

**Comment by wrexxor** [comment]
You're right on the money, the Tello is air cooled so it will shut down due to overheating if idle too long. Placing it in front of, or on top of, a fan can help keep it cool
